python/ray/ml/examples/rl_serving_example.py

```diff
@@ -0,0 +1,97 @@
+import argparse
+import gym
+import numpy as np
+
+import ray
+from ray.ml.checkpoint import Checkpoint
+from ray.ml.config import RunConfig
+from ray.ml.predictors.integrations.rl.rl_serve_env import RLServeEnv
+from ray.ml.predictors.integrations.rl.rl_predictor import RLPredictor
+from ray.ml.tests.test_rl_predictor import _DummyPolicy
+from ray.ml.train.integrations.rl.rl_trainer import RLTrainer
+from ray.ml.result import Result
+from ray.rllib import RolloutWorker
+from ray.serve.model_wrappers import ModelWrapperDeployment
+from ray import serve
+from ray.tune.tuner import Tuner
+
+
+def train_rl_ppo_online(num_workers: int, use_gpu: bool = False) -> Result:
```

Comment:
as an example for serving, should we just add a checkpoint to the codebase, and show only the serving side of story?

Reply:
Since our checkpoints are not guaranteed to be forward compatible I'd like to have this as an end to end example - it's mostly for users reading through it, and they should have an idea on how to generate a "matching" run without having to consult other sources

Reply:
ok, this is good for now.
connectors will help solve the forward compatibility problem. and we can revisit then.